packages/cryptoplease/lib/presentation/conversion_rates.dart

```diff
@@ -25,6 +25,7 @@ extension ConversionRates on BuildContext {
     required FiatCurrency fiatCurrency,
     required Token token,
     required int amount,
+    int? fiatDecimals,
```

Comment:
Shouldn't be like that. We have `fiatCurrency`, so decimals should be taken from it. There should be no situation when these values are out of sync.